test/game_test.rb

```diff
@@ -4,14 +4,14 @@
 class GameTest < Minitest::Test
   def setup
     info = {
-            game_id: 2012030221,
-            season: 20122013,
+            game_id: '2012030221',
+            season: '20122013',
             type: 'Postseason',
             date_time: '5/16/13',
-            away_team_id: 3,
-            home_team_id: 6,
-            away_goals: 2,
-            home_goals: 3,
+            away_team_id: '3',
```

Comment:
made these all strings to test that they were being converted to integers where I needed them to